src/elements/emby-button/emby-button.css

```diff
@@ -101,13 +101,12 @@
     min-height: initial;
     width: auto;
     height: auto;
-    padding: .556em;
+    padding: 0.556em;
     vertical-align: middle;
     border: 0;
-    vertical-align: middle;
+
     /* These are getting an outline in opera tv browsers, which run chrome 30 */
     outline: none !important;
-    position: relative;
```

Comment:
and why is this gone?

Reply:
Duplicate property, the other is hidden by Github further up.